openedx/core/djangoapps/fellow_auth0/middleware.py

```diff
@@ -155,8 +163,25 @@ def _authenticate_token(self, token):
                 f"User with email '{email}' does not exists on OpenEdx DB "
             )
             # create the user if it doesn't exist in the openedx db
-            password = self._get_random_password()
-            training_user = get_user_model().objects.create(username=name, email=email, password=password)
+            training_user = get_user_model().objects.create(username=username, email=email,is_active=True)
+            training_user.set_unusable_password()
```

Comment:
When we were using the previous password, the admin dashboard was stating that it couldn't figure out the generated password hash, I found this line in another part of the code and  to make it standard I figured that it may fix the issue that I'm having with the generated user.